composer/core/logging/logger_destination.py

```diff
@@ -63,3 +64,40 @@ def log_data(self, state: State, log_level: LogLevel, data: LoggerDataDict):
         """
         del state, log_level, data  # unused
         pass
+
+    def log_file_artifact(
```

Comment:
Would you be OK with shortening this to `log_artifact`?

Reply:
See below